webpack.config.js

```diff
@@ -1,36 +1,40 @@
+const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const ImagesConfigWebpackPlugin = require('image-config-webpack-plugin');
 const JsConfigWebpackPlugin = require('js-config-webpack-plugin');
 const ScssConfigWebpackPlugin = require('scss-config-webpack-plugin');
-const path = require('path');
 
 module.exports = {
-  entry: ['@babel/polyfill', './src/index.js'],
-  resolve: {
-    modules: ['node_modules'],
-  },
+  entry: './src/index.js',
   output: {
     path: path.join(__dirname, '/dist'),
-    filename: 'app.js',
+    filename: 'bundle.js',
     publicPath: '/',
   },
   module: {
     rules: [
       {
-        test: /\.m?js$/,
-        exclude: /(node_modules|bower_components)/,
-        use: {
-          loader: 'babel-loader',
-          options: {
-            presets: ['@babel/preset-env'],
-          },
+        test: /\.(css|mjs|js|jsx)$/i,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+
+        options: {
+          presets: [
+            '@babel/preset-env',
+            {
+              plugins: ['@babel/plugin-proposal-class-properties'],
```

Comment:
to keep code clean, move this plugin in the array of plugins at the bottom.

Reply:
cool